### ISSUE TITLE
Black text for dropdowns

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Permissions#2175]
I think we need black text for accessibility. I'm using this tool: https://accessibleweb.com/color-contrast-checker/

Red text fails:

<img width="1180" height="425" alt="Image" src="https://github.com/user-attachments/assets/cb1c70ab-b5e5-4aae-9c49-837dfde3e85d" />

Black text passes:

<img width="1165" height="406" alt="Image" src="https://github.com/user-attachments/assets/77b2f52a-561b-4a03-a893-f531449bd2d2" />
